### PR TITLE
vpnLookOut@claudiux: request to merge

### PR DESCRIPTION
**Pull request by claudiux**
Hi,

The vpnLookOut@claudiux is an applet that watches over the VPN link, displays its status, allows to reconnect the VPN link if it incidentally drops, and can stop or restart Transmission according to the state of the VPN.

All features are described in the README.md file.

Can you review it and tell me what changes I need to make, please?

If this applet is appropriate, can you merge its branch?

Many thanks.

Best regards,

claudiux

### CODE REVIEW

File: batterymonitor@pdcurtis/files/batterymonitor@pdcurtis/po/fr.po [hunk 1]
```diff
@@ -0,0 +1,194 @@
+# SOME DESCRIPTIVE TITLE.
```
Comment (jaszhix) [inline batterymonitor@pdcurtis/files/batterymonitor@pdcurtis/po/fr.po]: Can you add this PO file to a separate PR instead? Your commits will be squashed and we won't know from the commit history that this happened to a separate xlet. I think its okay if a PR is doing a similar thing to multiple xlets in individual commits and says so, but in this case its not related to adding the other applet.

File: vpnLookOut@claudiux/files/vpnLookOut@claudiux/applet.js [hunk 2]
```diff
@@ -0,0 +1,519 @@
...
+
+			// Now select icon to display
+			if (this.vpnStatus == "on") {
+				this.vpnIcon = this.vpnon ;
+				if (this.vpnName != "") {
+					this.button_connect.setStatus(_("Click to disconnect from VPN")+' '+this.vpnName);
+					this.button_connect.setToggleState(true)
+				}
+				this.alertFlag = false ;
+				vpnName = GLib.file_get_contents("/tmp/.vpn_name").toString().trim().substr(5);
```
[the remark this answers is not in the document]
Comment (jaszhix) [reply vpnLookOut@claudiux/files/vpnLookOut@claudiux/applet.js]: Also saw this:

```sh
Cjs-Message: JS LOG: this.button_connect is undefined
Cjs-Message: JS LOG: 
<----------------
anonymous/MyApplet.prototype.updateUI@/home/jason/.local/share/cinnamon/applets/vpnLookOut@claudiux/applet.js:439:6
anonymous/MyApplet.prototype.updateLoop@/home/jason/.local/share/cinnamon/applets/vpnLookOut@claudiux/applet.js:493:3
---------------->
```

**Comment by NikoKrause** [comment]
(1) Not a deal breaker, but could you add an English screenshot?

(2) Are the `apngframeXX.txt` really needed? They are all the same.

(3) You have a CHANGELOG.md and a changelog.txt file.

(4) If you want the README.md and the CHANGELOG.md to be shown on the spices website, they have to be in the same directory as the screenshot.png

File: batterymonitor@pdcurtis/files/batterymonitor@pdcurtis/po/fr.po (hunk 1 above)
Comment (claudiux) [reply batterymonitor@pdcurtis/files/batterymonitor@pdcurtis/po/fr.po]: Of course, that's a mistake, sorry. This file will be removed from this branch at the next commit.

File: vpnLookOut@claudiux/files/vpnLookOut@claudiux/applet.js [hunk 3]
```diff
@@ -0,0 +1,519 @@
...
+			// Finally setup to start the update loop for the applet display running
+			if (this.displayType == "compact") {
+				this.set_applet_label("");
+			} else {
+				this.set_applet_label("VPN");
+			}
+			this.set_applet_tooltip(_("Waiting"));
+			this.on_settings_changed()   // This starts the MainLoop timer loop
+
+		} catch (e) {
```
[the remark this answers is not in the document]
Comment (claudiux) [reply vpnLookOut@claudiux/files/vpnLookOut@claudiux/applet.js]: Corrected. These try-catch are commented ; there are only for debug.

**Comment by claudiux** [comment]
(1) An English screenshot replaces the French screenshot.
(2) The gif directory who contained the apngframeXX.txt is removed.
(3) changelog.txt is removed.
(4) README.md and CHANGELOG.md files moved.

Thanks!

File: vpnLookOut@claudiux/files/vpnLookOut@claudiux/applet.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (claudiux) [reply vpnLookOut@claudiux/files/vpnLookOut@claudiux/applet.js]: All these bugs are corrected. Many thanks!

**Comment by jaszhix** [comment]
Its working well for me now, nice work.

**Comment by NikoKrause** [comment]
Sorry, I'm afraid my English is a bit rusty. I still don't understand the screenshot:
https://github.com/linuxmint/cinnamon-spices-applets/blob/master/vpnLookOut%40claudiux/screenshot.png

**Comment by claudiux** [comment]
Sorry, I made it, but I forgot the git add command for this file. This will be in the next commit.

**Pull request merged**